mongodump/mongodump.go

```diff
@@ -38,22 +38,24 @@ type MongoDump struct {
 	InputOptions  *InputOptions
 	OutputOptions *OutputOptions
 
+	SkipUsersAndRoles bool
```

Comment:
To be clear, this isn't a configurable option in any way, so adding this field doesn't change the user experience. This is just a flag that mongomirror sets programmatically.